### PR TITLE
fix: neptune css fixes for angular components

### PR DESCRIPTION
**Pull request by dozortsev**
## ✅ Checklist

- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/master/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

File: packages/css/src/less/tick.less [hunk 1]
```diff
@@ -9,10 +9,13 @@
   &::after {
     content: '';
     position: absolute;
-    background: #fff;
+    background: var(--color-background-screen);
     transform: translateX(0.5px) rotate(-45deg);
     transform-origin: left bottom;
     left: 8px;
+    .tw-checkbox-button[disabled] & {
+      background-color: var(--color-text-inactive);
```
Comment (dozortsev) [inline packages/css/src/less/tick.less:17]: fix for checkbox https://github.com/transferwise/styleguide-components/pull/303#issuecomment-659515267

File: packages/css/src/less/core/_scaffolding.less [hunk 2]
```diff
@@ -335,8 +335,6 @@ object {
 
 img,
 video {
-  max-width: 100%;
-  height: auto;
```
Comment (dozortsev) [inline packages/css/src/less/core/_scaffolding.less:339]: fix for image inside "card" component
Screen Shot 2020-07-21 at 5 54 45 PM

File: packages/css/src/less/tick.less (hunk 1 above)
Comment (dozortsev) [reply packages/css/src/less/tick.less:17]: chosen color was approved by designer

File: packages/css/src/less/icons.less [hunk 3]
```diff
@@ -0,0 +1,20 @@
+@import './variables/_spacing.less';
+
+li,
+li > a > tw-icon:first-child {
```
Comment (StevePole) [inline packages/css/src/less/icons.less:4]: Are these styles specifically for the `tw-icon` angular component, or should this be `.tw-icon`?

**Comment by StevePole** [comment]
Can you add more screenshots of the styles applied to the angular components?

**Comment by dozortsev** [comment]
@StevePole 

`ngSelect` for regular icons

Screen Shot 2020-07-21 at 6 10 13 PM
Screen Shot 2020-07-21 at 6 10 04 PM


`ngSelect` for icons in circles

Screen Shot 2020-07-21 at 6 10 39 PM
Screen Shot 2020-07-21 at 6 10 24 PM

File: packages/css/src/less/icons.less (hunk 3 above)
Comment (dozortsev) [reply packages/css/src/less/icons.less:4]: yea, they are for angular `tw-icon` elements

**Pull request merged**